wallet/wallet-rpc-lib/src/config.rs

```diff
@@ -22,6 +22,18 @@ use common::chain::config::{
 use crypto::key::hdkd::u31::U31;
 use rpc::{rpc_creds::RpcCreds, RpcAuthData};
 
+#[derive(Clone)]
+pub enum NodeRpc {
+    ColdWallet,
+    HotWallet {
+        /// RPC address of the node to connect to
+        node_rpc_address: Option<String>,
+
+        /// Node RPC authentication
+        node_credentials: RpcAuthData,
```

Comment:
This should be `node_auth_data`... credentials mean something like username and password.

Reply:
renamed